teacher.rb

```diff
@@ -0,0 +1,15 @@
+require_relative './person'
+
+class Teacher < Person
+  def initialize(age, specialization, name = 'unknown', parent_permission: true)
+    @specialization = specialization
+    super(name, age, parent_permission: parent_permission)
+  end
```

Comment:
- [ ] The person class is accepting `age, name, and parent_permission` in the initializer and you are passing here `name, age, and parenent_persmission`, It will raise an error if you will try to create a teacher. It will store the name for age and age for the name. Please fix this issue. You can consider using this code `super(name, age, parent_permission: parent_permission)`